index.html

```diff
@@ -379,11 +462,41 @@ <h3>
     <p>
       When invoked, the <dfn class="export" data-dfn-for=
       "DigitalCredential">[[\Create]](origin, options,
-      sameOriginWithAncestors)</dfn> internal method MUST call the default
-      implementation of {{Credential}}'s {{Credential/[[Create]](origin,
-      options, sameOriginWithAncestors)}} internal method with the same
-      arguments.
+      sameOriginWithAncestors)</dfn> internal method MUST, if the user agent doesn't 
+      support issuance, call the default implementation of {{Credential}}'s 
+      {{Credential/[[Create]](origin,options, sameOriginWithAncestors)}} 
+      internal method with the same arguments. Otherwise, it MUST:
     </p>
```

Comment:
```suggestion
      sameOriginWithAncestors)</dfn> internal method, if the user agent doesn't 
      support issuance, call the default implementation of {{Credential}}'s 
      {{Credential/[[Create]](origin,options, sameOriginWithAncestors)}} 
      internal method with the same arguments. Otherwise:
    </p>
```